signals.mss

```diff
@@ -640,22 +644,35 @@ Format details:
       text-size: 10;
     }
 
-    /* can display Vr 1 */
+    /* DE: can display Vr 1                                          */
     /* This rule is overwritten if the signal can show Vr 2 as well. */
-    ["distant_states"=~"^(.*;)?DE-ESO:vr1(;.*)?$"] {
+    /* PL: can display Od2/Ot2  (expect clear)                       */
+    ["distant_states"=~"^(.*;)?DE-ESO:vr1(;.*)?$"],
+    ["distant_states"=~"^(.*;)?PL-PKP:od2(;.*)?$"],
+    ["distant_states"=~"^(.*;)?PL-PKP:ot2(;.*)?$"] {
```

Comment:
You need to add a `["feature"=…` for everything in here now. Before it were only German signals, now you could have mixed tagging. See below for main signals how this is done.

Reply:
should be good now